database/factories/ModelFactory.php

```diff
@@ -44,6 +44,7 @@
         'name' => $faker->company,
         'location' => $faker->postcode,
         'sponsor_id' => $sponsor_id, // a random sponsor
+        'payment_message' => $faker->sentence($nbWords = 12, $variableNbWords = true),
```

Comment:
I might prefix this with "Placeholder Payment Message: " or something. What do you think?